synapse/handlers/room.py

```diff
@@ -373,6 +375,15 @@ def send(etype, content, **kwargs):
                 content=content,
             )
 
+        if "encryption_alg" in config:
+            send(
+                etype="m.room.encryption",
```

Comment:
i think the standard in synapse is to put all event types in Types; otherwise it's ok?